src/wallet/wallet.cpp

```diff
@@ -1615,6 +1615,9 @@ bool CWallet::AddToWallet(const CWalletTx& wtxIn, bool fFromLoadWallet, CWalletD
         //// debug print
         LogPrintf("AddToWallet %s  %s%s\n", wtxIn.GetHash().ToString(), (fInsertedNew ? "new" : ""), (fUpdated ? "update" : ""));
 
+        // Call to balance slot
+        LogBalanceChanged();
```

Comment:
Call this `NotifyBalanceChanged`; maybe other listeners could do something other than logging.